public/app/plugins/panel/heatmap/HeatmapHoverView.tsx

```diff
@@ -237,7 +237,7 @@ const HeatmapHoverCell = ({ data, hover, showHistogram }: Props) => {
     <>
       <div>
         <div>{xDisp(xBucketMin)}</div>
-        <div>{xDisp(xBucketMax)}</div>
+        {!nonNumericOrdinalDisplay && <div>{xDisp(xBucketMax)}</div>}
```

Comment:
this is not good. it uses the condition for y bucket display to control x bucket display :(